<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>스팀 할인 목록</string>
  </property>
  <widget class="QComboBox" name="Discount_Rate">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>71</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Search_Start">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검색시작</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>타이틀</string>
   </property>
   <widget class="QListWidget" name="game_title_listbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>할인율(%)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MoveToWeb">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>22</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>스팀 홈페이지로 이동</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QLabel" name="Photo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Users/W15 208/Desktop/steam_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>현재날짜</string>
   </property>
   <widget class="QLabel" name="Datetime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="Dlc_list">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>DLC 목록[할인율, 원가, 할인가]</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>Discount_Rate</zorder>
  <zorder>Search_Start</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_2</zorder>
  <zorder>MoveToWeb</zorder>
  <zorder>Clear_button</zorder>
  <zorder>Photo</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>Dlc_list</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
